daemon/api_download.go

```diff
@@ -37,8 +37,9 @@ var snapDownloadCmd = &Command{
 
 // SnapDownloadAction is used to request a snap download
 type snapDownloadAction struct {
-	Action string   `json:"action"`
-	Snaps  []string `json:"snaps,omitempty"`
+	Action  string                `json:"action"`
+	Snaps   []string              `json:"snaps,omitempty"`
```

Comment:
not new in this PR but why does this have multiple snaps and action at all?

Reply:
there was no strong reason; I don't particularly like it either. I was close to flagging it in the original PR but it is the more generic solution so I could see where the author was coming from.

I'm ok with dropping them in the name of YAGNI.

Reply:
This is now updated based on https://github.com/snapcore/snapd/pull/7560